acceptance-tests/build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
   testRuntime 'javax.activation:activation'
   testRuntime 'org.apache.logging.log4j:log4j-core'
 
-  testImplementation project(':ethsigner')
+  testImplementation project(':ethsigner:app')
```

Comment:
Why add the `:app`? ...as you mentioned in a comment, adding to the `core` get's it everywhere, then why is a dependency on the app needed here?

Reply:
removed.